src/compat/util/over.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { describe, expect, it } from 'vitest';
+import { over } from './over';
+
+describe('over', () => {
+  it('should create a function that invokes `iteratees`', () => {
+    const func = over([Math.max, Math.min]);
```

Comment:
```suggestion
    const func = over(Math.max, Math.min);
```

lodash 테스트 케이스에서는 배열로 감싸지 않았다! 이부분 확인해주세요. 영어로 번역